examples/advanced_circuits_algorithms/Deutsch_Jozsa_Algorithm/Deutsch Jozsa Algorithm.ipynb

```diff
@@ -0,0 +1,1037 @@
+{
```

Comment:
<p>black box problem, where the task at hand is trying to determine what the black box is.</p><p> </p><p>applicable in real life, but showcases...</p><p> </p><p>computes a Boolean function, which only...</p><p> </p><p>what type of function U_f is based on only...</p><p> </p><p>is as large as 2^n, then the amount...</p><p> </p><p>large enough n, this problem...</p><p> </p><p>However, through leveraging a quantum algorithm, we only...</p><p> </p><p>different answers, each of which solves the problem.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/177/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>